include/wil/resource.h

```diff
@@ -4233,6 +4233,14 @@ namespace wil
 #if !defined(NOWINABLE)
     typedef unique_any<HWINEVENTHOOK, decltype(&::UnhookWinEvent), ::UnhookWinEvent> unique_hwineventhook;
 #endif
+#if !defined(NOCLIPBOARD)
+    using unique_clipboard = unique_call<decltype(::CloseClipboard()), &::CloseClipboard>;

```

Comment:
This should be `decltype(::CloseClipboard)` (not the result of the call)

Reply:
Seems all of the existing `unique_call` aliases are named more of the form `unique_close_clipboard_call`. Probably good to maintain consistency.